README.md

```diff
@@ -43,7 +43,7 @@ var data = d3.csvParse(string);
 
 ## API Reference
 
-<a name="csvParse" href="#csvParse">#</a> d3.<b>csvParse</b>(<i>string</i>[, <i>row</i>]) [<>](https://github.com/d3/d3-dsv/blob/master/src/csv.js "Source")
+<a name="csvParse" href="#csvParse">#</a> d3.<b>csvParse</b>(<i>string</i>[, <i>row</i>]) [<>](https://github.com/d3/d3-dsv/blob/master/src/csv.js "Source") :warning: [CSP in browser context](#content-security-policy)
 
 Equivalent to [dsvFormat](#dsvFormat)(",").[parse](#dsv_parse).
```

Comment:
```suggestion
Equivalent to [dsvFormat](#dsvFormat)(",").[parse](#dsv_parse). Note: requires unsafe-eval [content security policy](#content-security-policy).
```

Reply:
I've added the `Note` in the two places. (I was not sure if you wanted I delete the warning, so I didn't)

Reply:
Yes, please delete the warning (in favor of the note).

Reply:
Done.
Now we have:
- a note for csvParse and tsvParse
- A warning for dsv_parse